LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/MessageDispatcher.cs

```diff
@@ -51,15 +51,15 @@ public void DispatchRequest(LoRaRequest request)
 
             if (this.loraRegion == null)
             {
-                if (!RegionFactory.TryResolveRegion(request.Rxpk))
+                if (!RegionManager.TryResolveRegion(request.Rxpk))
```

Comment:
I would recommend using the overloaded that is not based on the static. This way we can one day remove the static `CurrentRegion`.

Reply:
Do you mean request.Region ?

Reply:
Yes, set request.Region to a local variable instead of static Current.